### ISSUE TITLE
[Bug]: When using SubprocVecEnv for parallel training of agents, the ep_rew_mean is no longer being recorded.

### ISSUE THREAD

**Issue by didu11** [DLR-RM/stable-baselines3#2027]
### 🐛 Bug

Hello, 

I have currently customized an environment using the gym library to train a robot for path tracking, and I am using the PPO algorithm from stable_baselines3 for training. 

My issue is that when I train the agent in a single environment, I can see the information related to "ep_len_mean" and "ep_rew_mean" in the terminal and in TensorBoard. However, when I use SubprocVecEnv to train the agent in three parallel environments (with the only difference being the trajectories give to the agent to trace, while all other settings are the same), the terminal and TensorBoard no longer display "ep_len_mean" and "ep_rew_mean." I would like to know why this is happening and how I can see "ep_len_mean" and "ep_rew_mean" when using SubprocVecEnv for parallel training. Thank you!

### To Reproduce

import gym
import torch
from stable_baselines3 import PPO
from stable_baselines3.common.vec_env import SubprocVecEnv
from Swimme_horizon import SwimmerEnvH
from Swimmer_incline import SwimmerEnvI
from Swimmer_vertical import SwimmerEnvV

import numpy as np
from stable_baselines3.common.callbacks import BaseCallback

def make_env(env_class, seed):
 def _init():
     env = env_class()
     env.seed(seed)
     return env
 return _init

if __name__ == '__main__':

 env_classes = [SwimmerEnvV, SwimmerEnvI, SwimmerEnvH]

 env_fns = [make_env(env_class, seed=0) for env_class in env_classes]
 vec_env = SubprocVecEnv(env_fns)

 model = PPO("MlpPolicy", vec_env, seed=0, verbose=1,
             tensorboard_log='....',
             learning_rate=2e-5,
             ent_coef=0.01,
             clip_range=0.2,
             batch_size=256,
             policy_kwargs={'net_arch': [256, 256]})

 model.learn(total_timesteps=int(2e6), tb_log_name='PPO')

 model.save("....")
 del model 

### Relevant log output / Error message

```shell
-------------------------------------------
| time/                   |               |
|    fps                  | 1367          |
|    iterations           | 2             |
|    time_elapsed         | 8             |
|    total_timesteps      | 12288         |
| train/                  |               |
|    approx_kl            | 0.00023688492 |
|    clip_fraction        | 0             |
|    clip_range           | 0.2           |
|    entropy_loss         | -7.1          |
|    explained_variance   | -0.00182      |
|    learning_rate        | 2e-05         |
|    loss                 | 121           |
|    n_updates            | 10            |
|    policy_gradient_loss | -0.000746     |
|    std                  | 1             |
|    value_loss           | 339           |
-------------------------------------------
```

### System Info

- OS: Linux-5.15.0-124-generic-x86_64-with-glibc2.17 # 134~20.04.1-Ubuntu SMP Tue Oct 1 15:27:33 UTC 2024
- Python: 3.8.19
- Stable-Baselines3: 2.3.2
- PyTorch: 1.13.0+cu117
- GPU Enabled: True
- Numpy: 1.24.4
- Cloudpickle: 3.0.0
- Gymnasium: 0.29.1
- OpenAI Gym: 0.21.0

### Checklist

- [x] My issue does not relate to a custom gym environment. (Use the custom gym env template instead)
- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have provided a [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014) example to reproduce the bug
- [X] I've used the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
you are missing monitor wrapper (see docs and other issues).
